src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java

```diff
@@ -35,8 +35,8 @@ public static void main(String[] args) {
      * @param depositAnnualPercentage ежегодный простой процент
      */
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        float percent = (float) depositAnnualPercentage / 100;
+        float percentTotal = (depositAmount * depositYears * percent);
+        return percentTotal + depositAmount;
```

Comment:
Invalid variable name because you cannot add a percentage to the amount. This is a wrong business transaction.